building on catalina 10.15.7 fails

im trying to build the project from the source code, as the precompiled binaries does not start (i.e. crashes in the opening process). I fixed some errors caused by missing libraries by installing them with home-brew. 
however, there seems to be something else here: 

`qhimdmainwindow.cpp:292:43: error: use of undeclared identifier 'qVariantFromValue'
        ui->himd_devices->addItem(device, qVariantFromValue((void *)dev));
                                          ^
1 error generated.
make[1]: *** [qhimdmainwindow.o] Error 1
make: *** [sub-qhimdtransfer-make_first] Error 2`

Do you recognize the error or could maybe track it to a further missing dependency? 

to reproduce this I 

1. clone the code in to a local directory
2. qmake
3. make

Thanks so much!

--- a/qhimdtransfer/qhimdmainwindow.cpp
+++ b/qhimdtransfer/qhimdmainwindow.cpp
@@ -289,7 +289,7 @@ void QHiMDMainWindow::device_list_changed(QMDDevicePtrList dplist)
     foreach(dev, dplist)
     {
         device = QString(dev->deviceType() == NETMD_DEVICE ? dev->name() : dev->name() + " at " + dev->path() );
-        ui->himd_devices->addItem(device, qVariantFromValue((void *)dev));
+        ui->himd_devices->addItem(device, QVariant::fromValue((void *)dev));
     }
 
     if(current_device)

--- a/qhimdtransfer/qmdmodel.cpp
+++ b/qhimdtransfer/qmdmodel.cpp
@@ -26,11 +26,11 @@ QVariant QNetMDTracksModel::headerData(int section, Qt::Orientation orientation,
         switch((ncolumnum)section)
         {
             case CoId:
-                return QSize(met.width("9999")+5, 0);
+                return QSize(met.horizontalAdvance("9999")+5, 0);
             case CoGroup:
             case CoTitle:
             case CoLength:
-                return QSize(met.width("9:99:99"), 0);
+                return QSize(met.horizontalAdvance("9:99:99"), 0);
             case CoCodec:
             case CoUploadable:
                 /* Really use the header for the metric in these columns,
@@ -175,20 +175,20 @@ QVariant QHiMDTracksModel::headerData(int section, Qt::Orientation orientation, 
         switch((hcolumnum)section)
         {
             case ColId:
-                return QSize(met.width("9999")+5, 0);
+                return QSize(met.horizontalAdvance("9999")+5, 0);
             case ColTitle:
             case ColArtist:
             case ColAlbum:
                 return QSize(25*met.averageCharWidth(), 0);
             case ColLength:
-                return QSize(met.width("9:99:99"), 0);
+                return QSize(met.horizontalAdvance("9:99:99"), 0);
             case ColCodec:
             case ColUploadable:
                 /* Really use the header for the metric in these columns,
                    contents will be shorter */
                 return QAbstractListModel::headerData(section,orientation,role);
             case ColRecDate:
-                return QSize(met.width("yyyy.MM.dd hh:mm:ss"), 0);
+                return QSize(met.horizontalAdvance("yyyy.MM.dd hh:mm:ss"), 0);
         }
     }
 

